src/common/components/header-with-content-link.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as styles from 'common/components/header-with-content-common.scss';
+import { NamedFC } from 'common/react/named-fc';
+import * as React from 'react';
+import { ContentLink, ContentLinkDeps } from 'views/content/content-link';
+import { ContentReference } from 'views/content/content-page';
+
+export interface HeaderWithContentLinkProps {
+    deps: ContentLinkDeps;
+    headerClass?: string;
+    headingTitleClassName?: string;
+    headingTitle: string;
+    secondaryText?: string;
+    guidance?: ContentReference;
+}
+
+export const HeaderWithContentLink = NamedFC<HeaderWithContentLinkProps>(
+    'HeaderWithContent Link',
```

Comment:
Looks like this extra space is propagating through to the snapshot markdown and looks odd. 
```suggestion
    'HeaderWithContentLink',
```

Reply:
Updated this in the last commit so resolving this